scripts/build.sh

```diff
@@ -193,63 +193,70 @@ if [[ $PLATFORM = Darwin || "$LIBGIT" < "24" ]] ; then
 	# feature). So we painfully need to build the dependencies ourselves.
 
 	if [ "$BUILD_DEPS" == "1" ] ; then
```

Comment:
so all of the changes in this commit are inside the `build-deps` condition. The commit message sounds like you are making changes for Travis, but of course this code isn't run on Travis.
I guess I'm confused...